src/modules/template-details/TemplateDetails.tsx

```diff
@@ -10,115 +10,114 @@ import ShareIcon from '@material-ui/icons/Share';
+            <ul className={[csx.basicList, csx.primary].join(' ')}>
+              {mapList(template.technologies)}
+            </ul>
           </span>
         </section>
 
         <section className={csx.details}>
           <span className={csx.watches}>
             <VisibilityIcon />
-            13k watch
+            {template.views}
           </span>
           <span className={csx.stars}>
             <StarBorderIcon />
-            3.5k stars
+            {template.stars}
           </span>
+          {/* MISSING FROM API? */}
           <p className={csx.createdBy}>Created 12 months ago by user</p>
         </section>
 
         <section>
           <h2 className={csx.header}>
-            <Button variant="icon" className={csx.button}>
-              <EditIcon />
-            </Button>
-            <span>React with model view provider</span>
+            <span>{template.name}</span>
           </h2>
 
-          <p className={csx.description}>
-            CSS uses a global namespace for CSS Selectors that can easily result in style conflicts
-            throughout your application when building an application using modern web components.
-            You can avoid this problem by nesting CSS selectors or use a styling convention like BEM
-            but this becomes complicated quickly and won’t scale. CSS-in-JS avoids these problems
-            entirely by generating unique class names when styles are converted to CSS. This allows
-            you to think about styles on a component level with out worrying about styles defined
-            elsewhere.
-          </p>
+          <p className={csx.description}>{template.description}</p>
         </section>
 
         <section className={csx.col}>
           <h3 className={csx.header}>
-            <Button variant="icon" className={csx.button}>
-              <EditIcon />
-            </Button>
             <span>Tech stack</span>
           </h3>
+          {/* MISSING FROM API? */}
           <ul className={[csx.basicList, csx.white].join(' ')}>{mapList(MOCKED_TECH_STACK)}</ul>
         </section>
 
         <section className={csx.col}>
           <h3 className={csx.header}>
-            <Button variant="icon" className={csx.button}>
-              <EditIcon />
-            </Button>
             <span>Patterns</span>
           </h3>
-          <ul className={[csx.basicList, csx.white].join(' ')}>{mapList(MOCKED_PATTERNS)}</ul>
+          <ul className={[csx.basicList, csx.white].join(' ')}>{mapList(template.patterns)}</ul>
         </section>
 
         <section className={csx.col}>
           <h3 className={csx.header}>
-            <Button variant="icon" className={csx.button}>
-              <EditIcon />
-            </Button>
             <span>Authors</span>
           </h3>
+          {/* MISSING FROM API? */}
           <ul style={{ display: 'inline-flex' }}>{mapImages(MOCKED_AUTHORS)}</ul>
```

Comment:
Avoid adding inline styles if you don't need. This makes investigation of styles much harder because always you must check inline def and css def